CMake/cdat_modules/pep8_pkg.cmake

```diff
@@ -0,0 +1,16 @@
+set( PEP8_MAJOR 1 )
+set( PEP8_MINOR 6 )
+set( PEP8_PATCH 2)
+set( PEP8_VERSION ${PEP8_MAJOR}.${PEP8_MINOR}.${PEP8_PATCH} )
+set( PEP8_URL ${LLNL_URL} )
+set( PEP8_GZ pep8-${PEP8_VERSION}.tar.gz )
+set( PEP8_MD5 c7a3f57d832484a6295164661fbb1335 )
+
+set (nm PEP8)
+string(TOUPPER ${nm} uc_nm)
+set(${uc_nm}_VERSION ${${nm}_MAJOR}.${${nm}_MINOR}.${${nm}_PATCH})
+set(PEP8_SOURCE ${PEP8_URL}/${PEP8_GZ})
+
+if (BUILD_TESTING)
```

Comment:
Right, there won't be any ctesting if it is OFF. I agree with you having testing ON by default is good. Having PEP installed when testing is ON is good IMO. If you agree, we can have it merged and then @dlonie can add style testing.